content/pages/burials-and-memorials/application/530.md

```diff
@@ -4,13 +4,15 @@ entryname: burials
 layout: page-react.html
 ---
 <div id="main">
-  <nav class="va-nav-breadcrumbs">
-    <ul class="row va-nav-breadcrumbs-list columns" role="menubar" aria-label="Primary">
+  <nav aria-label="Breadcrumb" aria-live="polite" aria-relevant="additions text" class="va-nav-breadcrumbs js-visual"
+  id="va-breadcrumbs">
+    <p class="usa-sr-only">Breadcrumb navigation will usually show all page links. It will adjust to show only the previous page when zoomed in, or viewed on a mobile device.</p>
+    <ol class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
       <li><a href="/">Home</a></li>
       <li><a href="/burials-and-memorials/"> Burials and Memorials </a></li>
       <li><a href="/burials-and-memorials/survivor-and-dependent-benefits/"> Survivors and Dependents </a></li>
-      <li><a href="/burials-and-memorials/survivor-and-dependent-benefits/burial-costs/">Handling Burial Costs (Allowances)</a></li>
-    </ul>
+      <li><a aria-current="page" href="/burials-and-memorials/survivor-and-dependent-benefits/burial-costs/">Handling Burial Costs (Allowances)</a></li>
```

Comment:
This last link isn't the page for the form, it's a content page that's one step back in the breadcrumbs.